docs/Migrating.md

```diff
@@ -15,6 +15,7 @@ The versions can be found in
 * The `play_against_current_self_ratio` self-play trainer hyperparameter has been renamed to `play_against_latest_model_ratio`
 * Removed the multi-agent gym option from the gym wrapper. For multi-agent scenarios, use the [Low Level Python API](Python-API.md).
 * The low level Python API has changed. You can look at the document [Low Level Python API documentation](Python-API.md) for more information. If you use `mlagents-learn` for training, this should be a transparent change.
+* The signature of `Agent.Heuristic()` was changes to take the `float[]` actions as a parameter, instead of returning the array. This was done to prevent a common source of error where users would return arrays of the wrong size.
```

Comment:
changes --> changed
the `float[]` --> `float[]` (?)

Reply:
Changed to
> The signature of `Agent.Heuristic()` was changed to take a `float[]` as a parameter...
(and in the changelog)